addons/l10n_ma/models/base_document_layout.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from markupsafe import Markup
+
+from odoo import api, fields, models
+
+
+class BaseDocumentLayout(models.TransientModel):
+    _inherit = 'base.document.layout'
+
+    @api.model
+    def _default_company_details(self):
+        company_data = {}
+        company_details = super()._default_company_details()
+        if self.env.company.country_code == 'MA':
+            company_data['l10n_ma_ice'] = self.env.company.l10n_ma_ice
+            company_details += Markup('<br> ICE: %(l10n_ma_ice)s') % company_data
```

Comment:
```suggestion
        company_details = super()._default_company_details()
        if self.env.company.country_code == 'MA':
            company_details += Markup('<br> ICE: %s', self.env.company.l10n_ma_ice)
```
Is this not simpler ?